Policy Set Exclusion: Product Docs (#431)
hashicorp/terraform-docs-common: website/docs/cloud-docs/policy-enforcement/manage-policy-sets.mdx

@include 'tfc-package-callouts/policies.mdx'
<!-- END: TFC:only name:pnp-callout -->

> Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the associated workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace. You can exclude selected workspaces from a policy set if you don't want those workspaces' runs to be checked against the policy set.

Author: "associated" didn't seem like quite the right word here, but I wanted something that meant essentially "For each run in the workspaces that the policy set applied to" -- e.g.:

global scoped -> all workspaces in the org
project scoped -> all workspaces in the project
workspace scoped -> all workspaces scoped directly

Reviewer: I think we've been using "attached" as the verb for "attaching a policy set to a project". also a small suggestion for active voice:

```suggestion
Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the attached workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace. You can exclude selected workspaces from a policy set if you don't want Terraform Cloud to check those workspaces' runs against the policy set.
```

Author: Thanks! I was trying to avoid "attached" because the sentence refers not just to runs on workspaces attached to the policy sets, but also runs in workspaces connected indirectly to a policy set (i.e. if the policy set is attached to the workspace's project or the policy set is global). Definitely a nit pick, so I'm happy to go back to "attached".

Author: Updated in bbdf764!

> Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the applicable workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace. You can exclude selected workspaces from a policy set if you don't want Terraform Cloud to check those workspaces' runs against the policy set.

Reviewer: I'm suggesting this shortened version because we do expand on this below, and this keeps the paragraph a bit slimmer:
```suggestion
Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the applicable workspaces, Terraform Cloud checks the Terraform plan against the policy set. Depending on the [enforcement level](#policy-enforcement-levels), failed policies can stop a run in a workspace. If you do not want to enforce a policy set on a specific workspace, you can exclude the workspace from that set.
```

Reviewer 2: 👋🏽 Jeff is now on PTO so I'm taking over this PR - that sounds good, Rose!

1. Choose a policy set scope:
   - **Policies enforced globally:** Terraform Cloud automatically enforces this global policy set on all of an organization's existing and future workspaces.
   - **Policies enforced on selected projects and workspaces:** Use the text fields to find and select the workspaces and projects to enforce this policy set on. This affects all current and future workspaces for any chosen projects.

> 1. **(Optional)** Add **Policy exclusions** for this policy set. Specify any workspaces in the policy set's scope that the policy set should not be enforced on.

Reviewer: suggestion for active voice:
```suggestion
1. **(Optional)** Add **Policy exclusions** for this policy set. Specify any workspaces in the policy set's scope that Terraform Cloud will not enforce this policy set on.
```
